fulltext-search-firestore/functions/index.js

```diff
@@ -52,37 +52,37 @@ function getFirebaseUser(req, res, next) {
   console.log('Check if request is authorized with Firebase ID token');
 
   if (!req.headers.authorization
-      || !req.headers.authorization.startsWith('Bearer ')) {
+    || !req.headers.authorization.startsWith('Bearer ')) {
     console.error(
       'No Firebase ID token was passed as a Bearer token in the Authorization header.',
       'Make sure you authorize your request by providing the following HTTP header:',
       'Authorization: Bearer <Firebase ID Token>'
-    );
-    res.status(403).send('Unauthorized');
-    return;
-  }
-
-  let idToken;
-  if (
-    req.headers.authorization &&
-    req.headers.authorization.startsWith('Bearer ')
+      );
+  res.status(403).send('Unauthorized');
```

Comment:
I'm honestly not a fan of passing non-meaningful strings. I'd perfer `res.sendStatus(403)`